#request fields
string target_spice
sensor_msgs/Image mask
sensor_msgs/Image color_frame
---
#response fields
bool success
int8 idx